packages/devtools-reps/src/reps/tests/grip.js

```diff
@@ -272,16 +272,18 @@ describe("Grip - Object with uninteresting properties", () => {
     expect(getRep(object)).toBe(Grip.rep);
   });
 
-  it.skip("renders as expected", () => {
+  it("renders as expected", () => {
     // @TODO This is broken at the moment.
     // See https://bugzilla.mozilla.org/show_bug.cgi?id=1276376
     const renderRep = props => shallowRenderRep(object, props);
-    const defaultOutput = 'Object {c: "c", d: 1, a: undefined, more...}';
+    const defaultOutput = 'Object { a: undefined, c: "c", d: 1, … }';
```

Comment:
In the test, we only want to remove the `.skip` (so the test will run).
The test should stay the same, and we should fix the `Grip` rep instead :)

Reply:
oh that makes sense. Alright I must have gotten it wrong. Alright let me take a second look at why they fail.